Makefile

```diff
@@ -379,12 +379,12 @@ endif
 # The rootfs partition size is set to 512MB after 10.2.0 release (see commit 719b4d516)
 # Before 10.2.0 it was 300MB. We must maintain compatibility with older versions so rootfs size cannot exceed 300MB.
 # kubevirt and nvidia are not affected by this limitation because there no installation of kubevirt prior to 10.2.0
-# Nethertheless lets check for ROOTFS_MAXSIZE_MB not exceeding 450MB for kubevirt and ARM and 270MB for x86_64
+# Nethertheless lets check for ROOTFS_MAXSIZE_MB not exceeding 900MB for kubevirt and ARM and 270MB for x86_64
```

Comment:
Code below uses 450 for ARM so please update the comment.

Reply:
ok will do and amend the commit

Reply:
Done